backend/src/event/EventProtocolEmitter.ts

```diff
@@ -0,0 +1,54 @@
+import { EventEmitter } from 'events'
+import { Event } from '@/event/Event'
+import { backendLogger as logger } from '@/server/logger'
+// import { EventProtocolType } from './EventProtocolType'
+import { EventProtocol } from '@entity/EventProtocol'
+// import { getConnection } from '@dbTools/typeorm'
+import CONFIG from '@/config'
+
+class EventProtocolEmitter extends EventEmitter {
+  private events: Event[]
+
+  public addEvent(event: Event) {
+    this.events.push(event)
+  }
+
+  public getEvents(): Event[] {
+    return this.events
+  }
+
+  public isEnabled() {
+    logger.info(`EventProtocol - isEnabled=${CONFIG.EVENT_PROTOCOL_ENABLED}`)
+    return CONFIG.EVENT_PROTOCOL_ENABLED
+  }
+}
+export const eventProtocol = new EventProtocolEmitter()
+
```

Comment:
Why do we need the `EventEmitter`? We can do all this without the overhead of the emitter. Just call `writeEvent` when you want to log an event. I can see a possible use for applications outside of the server, like triggering events when a new community was found. But I suggest to use the event emitter only for this cases.

Reply:
the EventEmitter of the node-Modul events is deleted now.
But the warning still exists in the Tests...